ctapipe/io/simteleventsource.py

```diff
@@ -205,6 +206,22 @@ def __exit__(self, exc_type, exc_val, exc_tb):
     def close(self):
         self.file_.close()
 
+    @property
+    def is_simulation(self):
+        return True
+
+    @property
+    def datalevels(self):
+        return ('R0', 'R1', 'DL0')
```

Comment:
Instead of generic strings, would it be worth to have these as fixed enums?

Reply:
Then a developer has a reference location to specify the correct levels instead of guessing, and adds a place for documenting the levels in the context of ctapipe

Reply:
Good Idea